site/_posts/2017-09-29-miqldap-to-sssd.md

```diff
@@ -14,22 +14,24 @@ How does one use the *miqldap_to_sssd* conversion tool?
 **Contents:**
 ---------------------------------------------------------------------
 
-- **A brief Why before The How:**
-- **The How**
-- **Let's kick the tires**
-  + **Example 1:** The simple Case:
-  + **Example 2:** A case with a small complication:
-  + **Example 3:** A case with a different small complication:
-- **The other miqldap_to_sssd arguments**
-- **Debugging Failed Conversions**
-
-**A brief Why before The How:**
+- [**A brief Why before The How**](#a-brief-why-before-the-how)
+- [**The How**](#the-how)
+- [**Kicking the tires**](#kicking-the-tires)
+  + [**Example 1 The simple Case**](#example-1-the-simple-case)
+  + [**Example 2 A case with a small complication**](#example-2-a-case-with-a-small-complication)
+  + [**Example 3 A case with a different small complication**](#example-3-a-case-with-a-different-small-complication)
+- [**The other miqldap_to_sssd arguments**](#the-other-miqldap_to_sssd-arguments)
+- [**Customizing The Generated Configuration**](#customizing-the-generated-configuration)
+- [**Debugging Failed Conversions**](#debugging-failed-conversions)
+
+**A brief Why before The How**
 =====================================================================
 
 Why did the *miqldap_to_sssd* conversion tool come to be?
 
-ManageIQ (MiQ) is delivered with the built in LDAP client, *MiqLdap*. Maintaining our own LDAP
-client is unnecessary as there are multiple mechanisms built into the operating system that provide an interface to an LDAP directory.
+ManageIQ (MiQ) is delivered with the built in LDAP client, *MiqLdap*. There are multiple, more robust and
+customizable, mechanisms built into the operating system that provide an interface to an LDAP directory.
+The  *miqldap_to_sssd* conversion tool is intended to automate and simplify moving to them.
```

Comment:
to them is too generic,  we only move to the SSSD authentication mechanism.

Reply:
Good point! 👍 Thank you!